frontend/src/components/DeletedNote.tsx

```diff
@@ -0,0 +1,72 @@
+import styles from "../styles/DeletedNote.module.css";
+import stylesUtils from "../styles/utils.module.css";
+import { Card } from "react-bootstrap";
+import { DeletedNote as DeletedNoteModel } from "../models/deletedNote";
+import { MdDelete, MdOutlineSettingsBackupRestore } from "react-icons/md";
+
+function formatExpiresIn(deletedAt: string): string {
```

Comment:
I'd refactor this using dayjs or momentjs as they are more readable and less error-prone